contracts/Core/StakeManager.sol

```diff
@@ -436,6 +468,23 @@ contract StakeManager is Initializable, ACL, StakeStorage {
         return(numStakers);
     }
 
+    /// @return age of staker
+    function getAge(uint256 stakerId) external view returns(uint256) {
+        return stakers[stakerId].age;
+    }
+
+    /// @return maturity of staker
+    function getMaturity(uint256 stakerId) external view returns(uint256) {
+        uint256 index = stakers[stakerId].age/10000;
+
+        return maturities[index];
+    }
+
+    /// @return influence of staker
+    function getInfluence(uint256 stakerId)  external view returns(uint256) {
+        return stakers[stakerId].age * stakers[stakerId].stake;
```

Comment:
here

Reply:
yes. fixed in https://github.com/razor-network/contracts/pull/218/commits/f8ccf9ce00ecce17230bed1d2a9199464358e940